internal/resp/connection/connection.go

```diff
@@ -42,6 +42,13 @@ type RespConnection struct {
 
 	// Callbacks is a set of functions that are called at various points in the client's lifecycle.
 	Callbacks RespConnectionCallbacks
+
+	// Offset tracking:
+	// SentBytes is the number of bytes sent to the server.
```

Comment:
```suggestion
	// SentBytes is the number of bytes sent using this connection.
```

We don't specify "server vs. client" anywhere in this file